am/templates/gateway/gateway-configmap.yaml

```diff
@@ -15,6 +15,12 @@ data:
     # Gateway HTTP server
     http:
       port: {{ .Values.gateway.service.internalPort }}
+      cookie:
```

Comment:
All the cookie block should be wrapped by a condition other wise we may face to some issue during the generation if as there are no default cookie settings into the values.yaml.

```
{{- if .Values.gateway.http.cookie }}
  cookie:
    secure: ...
{{- end }}
```